gce-scripts/packages_to_install.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Installs all the necessary packages for pshtt to run.
+# Logs which package it is installing as well as it's success (0) or failure
```

Comment:
Is failure OK in this case or should the script error out if a package install fails? If so, perhaps it would be simpler to use `set -e` at the top of the file?